File: Structure-syntax/09 Array Strings/Java/Main.java
/*
* Java program
*/
public class Main {
  public static void main(String [] argv) {
     char [] Arr = new char [26];
     char C = 'a';
     System.out.println("");
     for (int I = 0; I < Arr.length; ++I) {
        Arr [I] = C;
        System.out.print (C);
        C = (char)((int)C + 1);
     }
     System.out.println("\n");
  }
}

File: Structure-syntax/09 Array Strings/C++/src/main.cpp
/*
 * C++ program
 */
#include <iostream>
using namespace std;
int main(int argc, const char* argv[])  {
   char Arr [26];
   char C = 'a';
   cout << endl;
   for (int I = 0; I < 26; ++I)
     {
       Arr[I] = C;
       cout << C;
       C = C + 1;
     }
   cout << endl;
   cout << endl;
}

